app/config/AppConfig.scala

```diff
@@ -27,15 +28,10 @@ import scala.concurrent.duration.FiniteDuration
 @Singleton
 class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig) {
 
-  private lazy val contactHost                  = config.getOptional[String]("contact-frontend.host").getOrElse("")
-  private lazy val contactFormServiceIdentifier = config.get[String]("appName")
-
```

Comment:
please add these both back in for now

Reply:
This can be removed also as the library reads it from the application.conf etc. As these are not used within the codebase, they can be removed.